axelrod/strategies/memoryone.py

```diff
@@ -146,7 +146,7 @@ def __init__(self, ep=0.05):
         self.set_four_vector(four_vector)
 
 
-class ZeroDeterminantPlayer(MemoryOnePlayer):
+class LRPlayer(MemoryOnePlayer):
```

Comment:
No no, this wasn't a review question. Was just curious and too lazy to click the link... :)